terraform/transform_import_state.go

```diff
@@ -274,15 +274,15 @@ func (n *graphNodeImportStateSub) Execute(ctx EvalContext, op walkOperation) (di
 	}
 
 	// EvalRefresh
-	evalRefresh := &EvalRefresh{
+	evalRefresh := &EvalRefreshRequest{
 		Addr:           n.TargetAddr.Resource,
 		ProviderAddr:   n.ResolvedProvider,
 		Provider:       &provider,
-		ProviderSchema: &providerSchema,
-		State:          &state,
-		Output:         &state,
+		ProviderSchema: providerSchema,
+		State:          state,
 	}
-	diags = diags.Append(evalRefresh.Eval(ctx))
+	state, refreshDiags := Refresh(evalRefresh, ctx)
```

Comment:
Putting this comment at the very end (of my read) but it applies throughout: In this Eval refactor, especially because I really dig the ergonomics of `terraform.Refresh` (although, couldn't it be `refresh` and not be exported?):

We could drop the `eval` names lingering around. Ex. why not have this type be `refreshRequest` which you then pass to the `refresh` method. General naming suggestion that likely applies throughout this PR.

Reply:
For sure, there's a mess of inconsistent function names at this point. I was going to wait until my last PR - the apply-related Eval()s is all I have left - so I can ensure consistency throughout, but let me make a pass now!